index.js

```diff
@@ -62,7 +120,192 @@ function createTaskCreatedAt(taskCreatedAt) {
 function createTaskCheckBox() {
   return `<input
             type="checkbox"
-            class="table-box-check-box"
             name="checkbox-done"
           /> `;
 }
+
+function sortTasks(typeOfSort) {
+  if (typeOfSort == "a-to-z") {
+    sortTasksAtoZ();
+  } else {
+    unsortTasks();
+  }
+}
+
+function sortTasksAtoZ() {
+  const collator = new Intl.Collator("en", {
+    numeric: true,
+  });
+
+  tasks = tasks.sort((randomTask1, randomTask2) =>
+    collator.compare(
+      randomTask1.title.toUpperCase(),
+      randomTask2.title.toUpperCase()
+    )
+  );
```

Comment:
Well done :tada:

Reply:
Thamnksssssss.
I love this part cause it was completely new for me